.nancy-ignore

```diff
@@ -1,15 +0,0 @@
-# Since nancy doesn't respect replace directive, this check will fail until
-# upstream tendermint depends on websocket@1.4.0. We force a fixed version of
-# the websocket in go.mod: https://github.com/oasisprotocol/oasis-core/issues/2931
-# https://ossindex.sonatype.org/vuln/5f259e63-3efb-4c47-b593-d175dca716b0
-5f259e63-3efb-4c47-b593-d175dca716b0
```

Comment:
`go list -json -m all` (the new way to use nancy with go modules) only lists `websocket@1.4.0` so this isnt needed anymore,